test/mpi/.gitignore

```diff
@@ -1113,7 +1079,6 @@
 /manual/singjoin
 /manual/spawntest_child
 /manual/spawntest_parent
-/manual/test-port
```

Comment:
this looks like its valid. another temporary file created by a test - `manyconnect.in`.